spec/sql_query_specs.jest.ts

```diff
@@ -360,7 +360,7 @@ describe("check replace with $columns and concat and ARRAY JOIN", () => {
     let templateSrv = new TemplateSrvStub();
     const adhocFilters = [
         {
-            key: "default.test_array_join_nested.JobName",
```

Comment:
hm, why you try to fix test instead of figure out why test fail in your branch?
default.flows_raw.SrcAs specified and expected behavior here

Reply:
You were right, there is a bug in code that fails this test, will fix it